### PR TITLE
Getting Started Guide updates, round 1

### PR DESCRIPTION
**Pull request by zwalchuk**
A first round of updates for the Getting Started guides, focusing on reducing unnecessary text, standardizing language/formatting, updating screenshots, and improving navigation. Next round will include more thorough look at the deploying code section, as well as splitting off everything after the code download step into separate pages.

### CODE REVIEW

File: shared/flashingOsToBootMedia/_default.md [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
+1. Click *Select image* and find your application's **resinOS** image file.
+2. If you haven't already done so, insert your {{ $device.bootMedia }} into your computer. **Etcher** will automatically detect it. If you have more than one {{ $device.bootMedia }} inserted, you will need to select the appropriate one.
+3. Click the *Flash!* button.
 
 <img src="/img/common/etcher/etcher.gif" width="60%">
 
-Etcher will now prepare a bootable {{ $device.bootMedia }} and validate that it was flashed correctly. Right! time for a spot of tea as flashing the {{ $device.bootMedia }} can take roughly 3 or more minutes depending on the quality of your {{ $device.bootMedia }}.
-
-Etcher will give you a little ping! when it's done, and safely eject the {{ $device.bootMedia }} for you.
+**Etcher** will now prepare a bootable {{ $device.bootMedia }} and validate that it was flashed correctly. This can take roughly 3 or more minutes depending on the quality of your {{ $device.bootMedia }}. You'll a little ping when it's done, and **Etcher** will safely eject the {{ $device.bootMedia }} for you.
```
Comment (lurch) [inline shared/flashingOsToBootMedia/_default.md]: Looks like there's a word missing in "You'll a little ping"

File: shared/flashingOsToBootMedia/_default.md [hunk 2]
```diff
@@ -1,27 +1,16 @@
-Now we have to flash the downloaded `.img` file onto our {{ $device.bootMedia }}. We recommend using [Etcher][etcher-link], a simple, cross platform {{ $device.bootMedia }} writer and validator. Head over to the [Etcher homepage][etcher-link] and get install it, it only takes a few seconds :)
+The next step is to flash the downloaded image onto your {{ $device.bootMedia }} using [**Etcher**][etcher-link], a simple, cross platform {{ $device.bootMedia }} writer and validator. Once you have **Etcher** installed, start it up. To give **Etcher** access to your {{ $device.bootMedia }}, your system may prompt you to grant administrative privileges.
 
-You can of course use any other {{ $device.bootMedia }} writing software you like, some options are:
-* [win32diskimager][win32-disk-imager] for Windows.
-* [piFiller][pifiller-download] for osx.
-* [dd or "Disk Destroyer"][dd-link] for Linux.
```
Comment (lurch) [inline shared/flashingOsToBootMedia/_default.md]: For tidiness, I guess some of these link-URLs can now be removed from the footer-section of this page.

File: shared/getDeviceOnDash/artik10.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
-In order to get the resinOS up and running on our {{ $device.name }} we need to first set it up to boot from our {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the tiny `SW2` dip switch to position `1:on` and `2:on`, as shown below.
-
-__Note:__ This resinOS will completely write over the existing eMMC.
+__Note:__ **ResinOS** will completely write over the existing eMMC.
 
 <img src="/img/artik10/artik10-dev-kit.png" width="80%">
 
-We can now insert the 5VDC power cable and flip the power, switch labelled `PSW1`, to the `on` position. We should now have some glowing LEDs indicating a sign of life. Next we need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
+We can now insert the 5VDC power cable and flip the power, switch labelled `PSW1`, to the `on` position. You should now have some glowing LEDs indicating a sign of life. Next, you need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
```
Comment (lurch) [inline shared/getDeviceOnDash/artik10.md]: For consistency, I guess the first "We" on this line should be changed to "You" too.

File: shared/getDeviceOnDash/artik10.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+After a few seconds you should see your {{ $device.name }} appear on the **resin.io** dashboard. If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. 
 
 __Note:__ In order for the wifi to work correctly, you need to set jumpers `J20` and `J36` towards the edge of the board.
 
-Your {{ $device.name }} will now flash resinOS onto the internal eMMC so that you can remove the {{ $device.bootMedia }}. This will take 2 or 3 minutes, so time for more tea!! Once it has finished it will shut itself down and you will see the device on the dashboard in a `Post-Provisioning` state. At this point you need to:
+Your {{ $device.name }} will now flash **resinOS** onto the internal eMMC so that you can remove the {{ $device.bootMedia }}. This will take 2 or 3 minutes. Once it has finished it will shut itself down and you will see the device on the dashboard in a `Post-Provisioning` state. At this point you need to:
 1. Flip `PSW1` back to `off` position.
 2. Remove the {{ $device.bootMedia }}.
 3. set the  `SW2` dip switches to `1:off` and `2:off`.
```
Comment (lurch) [inline shared/getDeviceOnDash/artik10.md]: nit: Capital 'S' on 'set' ?

File: shared/getDeviceOnDash/artik5.md [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
+You can now insert the 5VDC power cable and flip the power, switch labelled `PWR SW`, to the `on` position. You should now have some glowing LEDs indicating a sign of life. Next, you need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
 
-After a few seconds you should see your {{ $device.name }} appear on the resin.io dashboard. If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. If you still can't get it online, contact us on support.
+After a few seconds you should see your {{ $device.name }} appear on the **resin.io** dashboard. If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. If you still can't get it online, contact us on support.
```
Comment (lurch) [inline shared/getDeviceOnDash/artik5.md]: The "If you still can't get it online, contact us on support." was removed in `artik10.md`, so should probably be removed here too?

File: shared/getDeviceOnDash/artik710.md [hunk 6]
```diff
@@ -1,10 +1,8 @@
-## Setting Up Your Device
-
-In order to get resinOS up and running on our {{ $device.name }} we need to first set it up to boot from our {{ $device.bootMedia }} rather than its 
-internal [eMMC memory][emmc-link]. To do this we need to set the `SW402` dip switch to position `1:off`, `2:off`, `3:off`, `4:on`, and insert the your {{ $device.bootMedia }}
+In order to get **resinOS** up and running on your {{ $device.name }}, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its 
+internal [eMMC memory][emmc-link]. To do this, you need to set the `SW402` dip switch to position `1:off`, `2:off`, `3:off`, `4:on`, and insert the your {{ $device.bootMedia }}
```
Comment (lurch) [inline shared/getDeviceOnDash/artik710.md]: "insert the your"?? Looks like "the" should be removed.

File: shared/getDeviceOnDash/beaglebone.md [hunk 7]
```diff
@@ -8,9 +6,9 @@ __Warning:__ This will completely overwrite any data on your devices' internal e
...
+After a short while you should see your device pop up in the dashboard. It will appear in a `configuring` state as it flashes **resinOS** to the internal media. This step can take a little time.
 
-After the internal media has been flashed, your device will shut itself down. At this point you will see the device in a "post-provisioning" state and all its LEDs should be off. Before booting the device again, make sure to **remove the SD card**. You may then simply press the power button situated nearest to the ethernet port or pull out and replug the power cable.
+After the internal media has been flashed, your device will shut itself down. At this point you will see the device in a `post-provisioning` state and all its LEDs should be off. Before booting the device again, *make sure to remove the SD card*. You may then simply press the power button situated nearest to the ethernet port or pull out and replug the power cable.
```
Comment (lurch) [inline shared/getDeviceOnDash/beaglebone.md]: Other guides have capitalised `post-provisioning` as `Post-Provisioning` ?
(only a small nit, but consistency is always good)

File: shared/getDeviceOnDash/hummingboard.md [hunk 8]
```diff
@@ -1,13 +1,11 @@
-## Setting Up Your Device
-
 Put the SD card into your device, and connect either the ethernet cable or WiFi adapter. Now power up the device by inserting the micro usb cable.
 
 It can take a few minutes for the device to boot up and appear on the dashboard, so grab some tea while you wait.
```
Comment (lurch) [inline shared/getDeviceOnDash/hummingboard.md]: In the other guides you've removed mentions of getting some tea.

File: shared/getDeviceOnDash/hummingboard.md [hunk 9]
```diff
@@ -1,13 +1,11 @@
-## Setting Up Your Device
-
 Put the SD card into your device, and connect either the ethernet cable or WiFi adapter. Now power up the device by inserting the micro usb cable.
```
Comment (lurch) [inline shared/getDeviceOnDash/hummingboard.md]: I'm obviously not familiar with how the substitutions / templating are being used in the docs, but should `SD card` here be `{{ $device.bootMedia }}` ? (same question applies to some of the other guides too)

File: shared/getDeviceOnDash/raspberrypi.md [hunk 10]
```diff
@@ -1,10 +1,8 @@
-## Setting Up Your Device
-
-Insert the SD card into the {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
+Insert the SD card into your {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
```
Comment (lurch) [inline shared/getDeviceOnDash/raspberrypi.md]: The `, if necessary,` here isn't in the raspberrypi2 guide? (or is this because the raspberrypi1 includes the PiZeroW, which has built-in Wi-fi?)

File: shared/getDeviceOnDash/raspberrypi.md [hunk 11]
```diff
@@ -1,10 +1,8 @@
...
 
 ![insert SD](/img/gifs/insert-sd.gif)
 
-It will take a minute or two for the device to appear on your [resin.io dashboard][resinDash]. While you wait the resinOS is expanding the partitions on your SD card to use all available space, installing a custom linux environment and establishing a secure connection with our servers.
+It will take a minute or two for the {{ $device.name }} to appear on your **resin.io** [dashboard][resinDash]. While you wait, the **resinOS** is expanding the partitions on your SD card to use all available space, installing a custom Linux environment, and establishing a secure connection with the **resin.io** servers.
```
Comment (lurch) [inline shared/getDeviceOnDash/raspberrypi.md]: I think the "expanding the partitions on your SD card to use all available space" sounds better than the "partitioning your SD card" that most of the other guides use :-)

File: shared/whatYouNeed/raspberrypi.md [hunk 12]
```diff
@@ -3,7 +3,7 @@
 * A 4GB or larger SD card. All the Raspberry Pi, except older model Bs, uses a microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
 * An ethernet cable or WiFi adapter (not needed for the Zero W). Check our [list of supported wifi adapters][wifiAdapters].
 * A micro USB cable.
-* **[Optional]** A [2A USB micro power supply][psu]
+* **[Optional]** A [2A USB micro power supply][psu].
```
Comment (lurch) [inline shared/whatYouNeed/raspberrypi.md]: nit: It's a "micro USB power supply" rather than a "USB micro power supply"

File: shared/whatYouNeed/raspberrypi.md [hunk 13]
```diff
@@ -3,7 +3,7 @@
 * A 4GB or larger SD card. All the Raspberry Pi, except older model Bs, uses a microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
 * An ethernet cable or WiFi adapter (not needed for the Zero W). Check our [list of supported wifi adapters][wifiAdapters].
```
Comment (lurch) [inline shared/whatYouNeed/raspberrypi.md]: WiFi adapter isn't needed for Raspberry Pi 3 either (like the Zero W, it has Wifi built-in)

File: shared/whatYouNeed/raspberrypi.md [hunk 14]
```diff
@@ -3,7 +3,7 @@
 * A 4GB or larger SD card. All the Raspberry Pi, except older model Bs, uses a microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
```
Comment (lurch) [inline shared/whatYouNeed/raspberrypi.md]: I think "all models of Raspberry Pi" might sound better than "All the Raspberry Pi" ? And "uses" should be "use".

File: shared/whatYouNeed/raspberrypi3.md [hunk 15]
```diff
@@ -1,8 +1,8 @@
 <img style="float: right;padding-left: 10px;" src="/img/raspberrypi/raspberrypi.jpg" width="25%">
 * A [Raspberry Pi 3 model B][rpi3B]. See our [supported devices list][supportedDevicesList] for other boards.
 * A 4GB or larger microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
-* **[Optional]** An ethernet cable
 * A micro USB cable.
+* **[Optional]** An ethernet cable.
 * **[Optional]** A [2A USB micro power supply][psu].
```
Comment (lurch) [inline shared/whatYouNeed/raspberrypi3.md]: Given the high-power-draw of the rpi3, perhaps we shouldn't label a 2A PSU as "optional" ?

File: pages/tools/cli.md [hunk 16]
```diff
@@ -461,7 +481,23 @@ confirm non interactively
...
+
+exact version number, or a valid semver range,
+or 'latest' (includes pre-releases),
+or 'default' (excludes pre-releases if at least one stable version is available),
+or 'recommended' (excludes pre-releases, will fail if only pre-release versions are available),
+or 'menu' (will show the interactive menu)
+
+#### --drive, -d &#60;drive&#62;
+
+the drive to write the image to, like `/dev/sdb` or `/dev/mmcblk0`. Careful with this as you can erase your hard drive. Check `resin util available-drives` for available options.
```
Comment (lurch) [inline pages/tools/cli.md]: Perhaps "Be careful with this..." would read better?

File: config/dictionaries/device.json [hunk 17]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "id": "raspberrypi",
-    "name": "Raspberry Pi 1, Zero, or Zero W",
+    "name": "Raspberry Pi (v1 or Zero)",
```
[the remark this answers is not in the document]
Comment (alisondavis17) [reply config/dictionaries/device.json]: Why remove the explicit reference to Zero W? I know we treat it the same as the Zero for all intents and purposes, but a lot of people ask(ed) about Zero W explicitly, so it's nice to have a callout

File: pages/runtime/supervisor-api.md [hunk 18]
```diff
@@ -459,8 +459,8 @@ Returns the application running on the device
 The app is a JSON object that contains the following:
 * `appId`: The id of the app as per the Resin API.
 * `commit`: Application commit that is running.
-* `imageId`: The Docker image of the current application build.
-* `containerId`: ID of the Docker container of the running app.
+* `imageId`: The docker image of the current application build.
```
Comment (alisondavis17) [inline pages/runtime/supervisor-api.md]: Why lowercase docker? Is it written as code?

File: pages/tools/cli.md [hunk 19]
```diff
@@ -4,7 +4,7 @@ This tool allows you to interact with the resin.io api from the comfort of your
 
 Please make sure your system meets the requirements as specified in the [README](https://github.com/resin-io/resin-cli).
 
-To get started, download the CLI from npm:
+To get started download the CLI from npm.
```
Comment (alisondavis17) [inline pages/tools/cli.md]: might be worth adding @pimterry as a reviewer for this section on CLI docs?

File: pages/runtime/supervisor-api.md (hunk 18 above)
Comment (alisondavis17) [reply pages/runtime/supervisor-api.md]: (same applies for a bunch of lowercase Docker instances below - if it's not written as a command or code, it needs to be uppercase)

Comment (alisondavis17) [reply pages/runtime/supervisor-api.md]: per Docker: 
>Capitalize the D in Docker, Dockerfile, Docker Swarm and other marks consisting of "Docker", except when you are specifically referring to the commandline command, for example for 'docker run'. Then you may use 'docker'.

File: pages/tools/cli.md (hunk 19 above)
Comment (lurch) [reply pages/tools/cli.md]: :+1:  When reviewing this, I glossed over most of the CLI and SDK docs ;-)

Comment (zwalchuk) [reply pages/tools/cli.md]: These get pulled in from the separate CLI repo, so any changes we need to make should be requested there. I'll be making a PR soon to make sure Docker is capitalized everywhere in that repo

File: shared/whatYouNeed/raspberrypi.md (hunk 12 above)
Comment (lurch) [reply shared/whatYouNeed/raspberrypi.md]: Also, why do we need "A micro USB cable."?
AFAICT the requirement is _either_ for a micro USB power supply like [this](https://www.amazon.co.uk/Micro-Supply-Charger-Raspberry-Pi/dp/B00AUKR4EU) (which has a moulded microUSB plug) **OR** a USB power supply like [this](https://www.amazon.co.uk/Technoplay-ltd-Mains-Charger-Charge/dp/B001SGXGGA) and a micro USB cable?

File: config/dictionaries/device.json (hunk 17 above)
[the remark this answers is not in the document]
Comment (zwalchuk) [reply config/dictionaries/device.json]: This was done to align with what we have on the dashboard dropdown. I'll talk to the frontend team about a consistent and not too wordy type name we could use across both

[the remark this answers is not in the document]
Comment (alisondavis17) [reply config/dictionaries/device.json]: Gotcha - yeah I think it would be worth calling out the W there as well

File: shared/createAnApp/_default.md [hunk 20]
```diff
@@ -1,5 +1,7 @@
-To create an application simply type in a name, select the {{ $device.name }} type from the drop down list and click the create button. You should now be taken to the dashboard of your newly created application:
+An application is a group of devices of the same type that all run the same code. When you provision a device, it is added to a specific application, but can be migrated to another application at any time.
 
-<img src="/img/common/main_dashboard/select_fleet_type.png" width="80%">
+To create an application, simply select the {{ $device.name }} type from the drop-down list, type in a name, and click *CREATE NEW APPLICATION*:
 
-This dashboard may not look like much right now, but this is where you will command and manage a whole fleet of {{ $device.name }}s.
+<img src="/img/common/main_dashboard/select_fleet_type.png" width="40%">
```
Comment (lurch) [inline shared/createAnApp/_default.md]: It would probably be a fair chunk of work creating all the screenshots, but it would be awesome if the image used here could also be paramaterised by `$device` so that e.g. the image at https://docs.resin.io/beaglebone/nodejs/getting-started/#creating-an-application could show the Beaglebone selected in the device-type dropdown :-)

Comment (zwalchuk) [reply shared/createAnApp/_default.md]: It shouldn't be too difficult. I'll see how easy it is to implement with the current set up, and it will definitely be this way once we have our docs generated by contracts.

File: pages/runtime/supervisor-api.md (hunk 18 above)
Comment (zwalchuk) [reply pages/runtime/supervisor-api.md]: Another one that is pulled in external. I'll make a PR to get these all fixed.

**Comment by lurch** [comment]
P.S. I've just noticed that near the top of https://docs.resin.io/raspberrypi/nodejs/getting-started/ there's a link that says "Looking for other boards documentation? Click here." which links to https://docs.resin.io/installing/otherboards/
Are the devices in that list going to be updated to support the same "templating" scheme as all the other devices' documentation?  At the very least, I strongly suspect that https://docs.resin.io/installing/otherboards/ **shouldn't** say "all the boards supported by resin.io" ??
